hledger-dep.json

```diff
@@ -0,0 +1,3386 @@
+{
+  "name": "hledger-exe",
+  "buildsystem": "simple",
+  "build-commands": [],
+  "modules": [
+    {
+      "name": "ghc-9.6.6-aarch64",
+      "only-arches": [
+        "aarch64"
+      ],
+      "cleanup": [
+        "/bin/ghc",
+        "/bin/ghc-9.6.6",
+        "/bin/ghc-pkg",
+        "/bin/ghc-pkg-9.6.6",
+        "/bin/ghci",
+        "/bin/ghci-9.6.6",
+        "/bin/haddock",
+        "/bin/haddock-ghc-9.6.6",
+        "/bin/runghc",
+        "/bin/runghc-9.6.6",
+        "/bin/runhaskell",
+        "/bin/hp2ps",
+        "/bin/hpc",
+        "/bin/hsc2hs",
+        "/lib/libtinfo.so.5",
+        "/lib/debug",
+        "/lib/ghc-9.6.6",
+        "/share/man",
+        "/share/doc/ghc-9.6.6"
+      ],
+      "buildsystem": "simple",
+      "builddir": false,
+      "build-commands": [
+        "mkdir -p /app/lib",
+        "ln -s /usr/lib/aarch64-linux-gnu/libtinfo.so /app/lib/libtinfo.so.5",
+        "./configure --prefix=/app",
+        "make install"
+      ],
+      "sources": [
+        {
+          "sha256": "58d5ce65758ec5179b448e4e1a2f835924b4ada96cf56af80d011bed87d91fef",
+          "type": "archive",
+          "url": "https://downloads.haskell.org/ghc/9.6.6/ghc-9.6.6-aarch64-deb10-linux.tar.xz"
+        }
+      ]
+    },
+    {
+      "name": "ghc-9.6.6-x86_64",
+      "only-arches": [
+        "x86_64"
+      ],
+      "cleanup": [
+        "/bin/ghc",
+        "/bin/ghc-9.6.6",
+        "/bin/ghc-pkg",
+        "/bin/ghc-pkg-9.6.6",
+        "/bin/ghci",
+        "/bin/ghci-9.6.6",
+        "/bin/haddock",
+        "/bin/haddock-ghc-9.6.6",
+        "/bin/runghc",
+        "/bin/runghc-9.6.6",
+        "/bin/runhaskell",
+        "/bin/hp2ps",
+        "/bin/hpc",
+        "/bin/hsc2hs",
+        "/lib/libtinfo.so.5",
+        "/lib/debug",
+        "/lib/ghc-9.6.6",
+        "/share/man",
+        "/share/doc/ghc-9.6.6"
+      ],
+      "buildsystem": "simple",
+      "builddir": false,
+      "build-commands": [
+        "mkdir -p /app/lib",
+        "ln -s /usr/lib/x86_64-linux-gnu/libtinfo.so /app/lib/libtinfo.so.5",
```

Comment:
this won't work on aarch64

Reply:
this ghc module is `"only-arches": ["x86-64"]`. the aarch64 version is the one above. i've built the app locally with `--arch=aarch64` and it worked, though i don't have a real aarch64 device to test on

Reply:
Ah I see it now